src/azure-cli/azure/cli/command_modules/iot/tests/hybrid_2020_09_01/test_iot_central_commands.py

```diff
@@ -0,0 +1,82 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from azure.cli.testsdk import ScenarioTest, ResourceGroupPreparer
+import unittest
+
+
+class IoTCentralTest(ScenarioTest):
+
+    @unittest.skip("Need to check this")
```

Comment:
Skipped tests

Reply:
iot test was taken from master, few of them is not supported in 2020-09-01-hybrid and hence it was marked for skipped.
i think the skip description is not very accurate. i will remove these test cases in next PR along with removing old profile test cases.